lib/ansible/module_utils/redfish_utils.py

```diff
@@ -692,6 +692,35 @@ def get_boot_order(self, systems_uri):
     def get_multi_boot_order(self):
         return self.aggregate(self.get_boot_order)
 
+    def get_boot_override(self):
+        result = {}
+
+        properties = ["BootSourceOverrideEnabled", "BootSourceOverrideTarget",
+                      "BootSourceOverrideMode", "UefiTargetBootSourceOverride"]
+
+        response = self.get_request(self.root_uri + self.systems_uris[0])
+        if response['ret'] is False:
+            return response
+        result['ret'] = True
+        data = response['data']
+
+        if 'Boot' not in data:
+            return {'ret': False, 'msg': "Key Boot not found"}
+
+        boot = data['Boot']
+
+        result['boot_override'] = {}
+        if "BootSourceOverrideEnabled" in boot:
+            if boot["BootSourceOverrideEnabled"] is not False:
```

Comment:
the value of this property is "Disabled", "Once" or "Continuous".. Maybe no need to check its value for the output

Reply:
I set it to check for being Disabled and returning that there is no override present if so.